docs/static/offline-plugins.asciidoc

```diff
@@ -1,17 +1,98 @@
 [[offline-plugins]]
 === Offline Plugin Management
 
-The Logstash <<working-with-plugins,plugin manager>> was introduced in the 1.5 release. This section discusses setting up
-local repositories of plugins for use on systems without access to the Internet.
+The Logstash <<working-with-plugins,plugin manager>> provides support for preparing offline plugin packs that you can
+use to install Logstash plugins on systems that don't have Internet access. 
 
-The procedures in this section require a staging machine running Logstash that has access to a public or private Rubygems
-server. This staging machine downloads and packages the files used for offline installation.
+This procedure requires a staging machine running Logstash that has access to a public or
+<<private-rubygem,private Rubygems>> server. The staging machine downloads and packages all the files and dependencies
+required for offline installation.
+
+//REVIEWERS: I'm assuming that a private gem repository is still an option, but I haven't tested it. Is it still an option?
```

Comment:
Yes it still an option, we didn't change anything for that, but the prepare-offline-pack should be easier to use than configuring a private gem server.